<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PKU-PreElective-v1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>未选课程列表：（双击选课）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="CourseTable">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>课名：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="CourseNameEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>开课单位：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="CollegeEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>课程类型：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="TypeEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>已选列表：（双击退课）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="SelectedTable">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="CreditLabel">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>当前总学分：0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="DisableColumn">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>隐藏课号、班号、起止周</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="TimeTableLayout">
        <item>
         <widget class="QScrollArea" name="ScrollFig">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>63</width>
             <height>159</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>冲突选课：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="CrashLabel">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>60</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
